Client/Frontend/Browser/TabManager.swift

```diff
@@ -500,7 +500,17 @@ class TabManager: NSObject {
         privateConfiguration = TabManager.makeWebViewConfig(isPrivate: true, prefs: profile.prefs)
     }
 
-    func removeTabsWithUndoToast(_ tabs: [Tab]) {
+    func removeTabsNoToast(_ tabs: [Tab]) {
+        for tab in tabs {
+            self.removeTab(tab, flushToDisk: false, notify: true)
```

Comment:
can you use `removeTabs` here for consistency with the toast remove function?

Reply:
@dnarcese I actually thought about using it but since we also need to add a tab `removeTabs` won't be helpful here.

Reply:
or you mean just change the name of the function 😅

Reply:
What do you mean?

Reply:
`removeTab` and `removeTabs` are two different methods that we already have in our class and both serve a different purpose. `removeTab` allows us to set flushing to disk / notifying of closing of tabs and `removeTabs` method uses `removeTab` and simply deletes all tabs and stores to disk

For our usecase we want to remove tabs but not flush to disk until we add a new one hence why we can't just use `removeTabs` method

The naming can be confusing so I was wondering if you want me to change the name?

Reply:
But doesn't `removeTabs` also do `self.removeTab(tab, flushToDisk: false, notify: true)`. It looks the same as your code unless I am missing something else

Reply:
Is it just because of when storeChanges() is called?

Reply:
Small difference though 
 
 `removeTabsNoToast`
 ```
// This part is same 
for tab in tabs {
     self.removeTab(tab, flushToDisk: false, notify: true)
}
// but it adds a new tab
if normalTabs.isEmpty {
     selectTab(addTab())
}
// once tab is added only then we make store changes
     storeChanges()
```

Where as for `removeTabs` we don't add a new tab

```
for tab in tabs {
     self.removeTab(tab, flushToDisk: false, notify: true)
}
// No new tab is added and we directly make store changes
storeChanges()
 
```

Reply:
Ok. So yes I think renaming it would be helpful